EpikCord/client/http.py

```diff
@@ -136,7 +160,10 @@ def __init__(self, token: str, *, version: int = 10):
             headers={
                 "Authorization": f"Bot {self.token}",
                 "User-Agent": f"DiscordBot (https://github.com/EpikCord/EpikCord.py {__version__})",
-            }
+            },
+            json_serialize=lambda x, *__, **___: json.dumps(x).decode("utf-8") # type: ignore
```

Comment:
what is this?

Reply:
This is the function used to serialize JSON when making a request. `orjson` returns bytes and so I need to get that back as a string using `.decode`, and then I just use `json` if `orjson` is not installed.